clients/admin-ui/src/features/user-management/UserManagementRow.tsx

```diff
@@ -123,46 +117,17 @@ const UserManagementRow: React.FC<UserManagementRowProps> = ({ user }) => {
         <Td pl={0} py={1} onClick={handleEditUser}>
           {user.created_at ? new Date(user.created_at).toUTCString() : null}
         </Td>
-        <Restrict
-          scopes={[
-            ScopeRegistryEnum.USER_UPDATE,
-            ScopeRegistryEnum.USER_DELETE,
-          ]}
-        >
+        <Restrict scopes={[ScopeRegistryEnum.USER_DELETE]}>
           <Td pr={0} py={1} textAlign="end" position="relative">
             <ButtonGroup>
-              <Menu>
-                <MenuButton
-                  as={Button}
-                  size="xs"
-                  bg="white"
-                  data-testid="menu-btn"
-                >
-                  <MoreIcon color="gray.700" w={18} h={18} />
-                </MenuButton>
-                <Portal>
-                  <MenuList shadow="xl" data-testid={`menu-${user.id}`}>
-                    <Restrict scopes={[ScopeRegistryEnum.USER_UPDATE]}>
-                      <MenuItem
-                        _focus={{ color: "complimentary.500", bg: "gray.100" }}
-                        onClick={handleEditUser}
-                        data-testid="edit-btn"
-                      >
-                        <Text fontSize="sm">Edit</Text>
-                      </MenuItem>
-                    </Restrict>
-                    <Restrict scopes={[ScopeRegistryEnum.USER_DELETE]}>
-                      <MenuItem
-                        _focus={{ color: "complimentary.500", bg: "gray.100" }}
-                        onClick={deleteModal.onOpen}
-                        data-testid="delete-btn"
-                      >
-                        <Text fontSize="sm">Delete</Text>
-                      </MenuItem>
-                    </Restrict>
-                  </MenuList>
-                </Portal>
-              </Menu>
+              <IconButton
```

Comment:
out of curiosity, does making this change allow us to put the `handleEditUser` on the whole row instead of the individual `<Td>s`?